scripts/generate-test-certs-task.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+docker run --rm -v "$PWD:/usr/src/app" -e "NODE_ENV=development" -w "/usr/src/app/certs" circleci/node:10.14.1 yarn generate:test-certs
```

Comment:
why we need to run the script in a docker container? can't we just run `yarn generate:test-certs` ?

Reply:
Yes, we can. Considering that the application runs in docker containers and we already use the `circleci/node:10.14.1` image, I thought to use the same image in order to run that script, so that there is no need to add `yarn` to the dependencies to run the application.